architecture/core_objects/routing.adoc

```diff
@@ -330,18 +340,18 @@ certificate that is provided in the base image.
 
 Currently, password protected key files are not supported. HAProxy prompts you
 for a password upon starting and does not have a way to automate this process.
-To remove a passphrase from a keyfile, you can run:
+To remove a passphrase from a keyfile, run:
 
-****
-`# openssl rsa -in _<passwordProtectedKey.key>_ -out _<new.key>_`
-****
+----
+# openssl rsa -in _<passwordProtectedKey.key>_ -out _<new.key>_
+----
 
```

Comment:
You can kill the underscores, as they are no longer markup, but instead literal text, in a ---- block (IIUC).